.github/workflows/CI.yml

```diff
@@ -31,3 +31,5 @@ jobs:
           -Dhome.application.admin.username=${{secrets.HOME_APPLICATION_ADMIN_USERNAME}}
           -Dhome.application.admin.password=${{secrets.HOME_APPLICATION_ADMIN_PASSWORD}}
           -Dapi.tests.verbose.logging=${{env.API_TESTS_VERBOSE_LOGGING}}
+          -Dgithub.packages.read.username=${{secrets.PACKAGES_READ_USERNAME}}
+          -Dgithub.packages.read.token=${{secrets.PACKAGES_READ_TOKEN}}
```

Comment:
Why are here two workflows with the same name?

Reply:
We need those two params a both of our workflows in order to get dependency from GitHub.

Reply:
We need those two params in both of our workflows in order to get dependency from GitHub.